modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogIndexDescriptor.java

```diff
@@ -35,11 +35,15 @@ public abstract class CatalogIndexDescriptor extends CatalogObjectDescriptor {
     /** Index status. */
     private final CatalogIndexStatus status;
 
-    CatalogIndexDescriptor(int id, String name, int tableId, boolean unique, CatalogIndexStatus status) {
+    /** Catalog version in which the index was created. */
+    private final int creationVersion;
```

Comment:
I would suggest to rename this parameter to `catalogCreationVersion`. We already have a mixup in "table versions" and "catalog version". Better be explicit

Reply:
Ok, why not.